pom.xml

```diff
@@ -63,6 +63,7 @@
         <aspectj.version>1.9.21.2</aspectj.version>
         <allure.version>2.25.0</allure.version>
         <allure.maven.version>2.12.0</allure.maven.version>
+        <test-frame.version>0.1.0-SNAPSHOT</test-frame.version>
```

Comment:
snapshot is here to stay?

Reply:
no :) I will switch that once I release version with stable changes, beta3 I would say

Reply:
```
 Error:  COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
Error:  /home/runner/work/odh-e2e/odh-e2e/src/test/java/io/odh/test/e2e/Abstract.java:[49,42] cannot find symbol
  symbol:   method addCreateCallback((r)->{ if [...]; } })
  location: class io.skodjob.testframe.resources.KubeResourceManager
```

in gha tests, that's something wrong in the test-frame lib itself? I can find no references to `addCreateCallback` here.

Reply:
fixed, due to snaphot was cachet I had to remove cache in GH action to download latest snapshot :)

Reply:
either that, or maybe give it `--update-snapshots` for now?

Reply:
I don not expect merge today :) I will release tomorrow or later this week, then switch to maven central here and then merge :)